website/screens/components/nav-tabs/usage/NavTabsUsagePage.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  DxcBulletedList,
+  DxcFlex,
+  DxcParagraph,
+  DxcLink,
+} from "@dxc-technology/halstack-react";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Image from "@/common/Image";
+import navTabsNavigation from "./images/navTabs_navigation.png";
+
+const sections = [
+  {
+    title: "Usage",
+    content: (
+      <>
+        <DxcParagraph>
+          Nav tabs organize and allow navigation across different pages of
+          content that are related and at the same level of hierarchy. They are
+          commonly used in the header of a page.
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            Use nav tabs for page or site navigation when dividing content into
+            different pages.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            The content should follow an organized structure and hierarchy.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Text labels should be short and have a clear relation to content.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Know when to use nav tabs for page navigation and regular tabs for
```

Comment:
I think this point is a bit redundant with the first one. Maybe we should rephrase it.

Reply:
Rephrased!